webapp/scripts/web-server.js

```diff
@@ -40,8 +40,12 @@ function HttpServer(handlers) {
 
 HttpServer.prototype.start = function(port) {
   this.port = port;
-  this.server.listen(port);
-  util.puts('Http Server running at http://localhost:' + port + '/');
+  console.log("Starting web server at http://localhost:" + port + "/");
+  this.server.listen(port).on('error', function(err) {
+    if (err.code === "EADDRINUSE") {
+      console.error("\033[31mPort %d is already in use, can't start web server.", port);
```

Comment:
This leaves the terminal color red. Could you put a '\033[0m' at the end of the error line to reset the terminal color?